.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## Description
+[Please provide general introduction to the issue/proposal.]
+
+
+## Reproduction
+[If possible, please include example code and image(s) to reproduce this Issue.]
+
+
+## Environment
+[If reporting a bug, please include the following important infomation about your system.]
+- [ ] Operating system and its version
+- [ ] Python version (run `sys.version`)
+- [ ] scikit-image version (run `skimage.__version__`)
+
+
+## Traceback
+[If reporting a bug, please paste the entire traceback from Python here.]
+
+
+## References
```

Comment:
I'm with Stéfan on this one. Combine and condense; if we don't need it literally every time we shouldn't ask for it.

This may result in the devs shouldering a bit more of the burden, but that's a price I'm willing to pay to ensure the barriers to making contributions stay as low as possible.